stdlib/public/core/UnsafePointer.swift

```diff
@@ -965,8 +1135,9 @@ public struct UnsafeMutablePointer<Pointee>: _Pointer {
     //   self[i] = (source + i).move()
     // }
   }
+}
 
-  @inlinable
+extension UnsafeMutablePointer {
   @_alwaysEmitIntoClient
   @available(*, deprecated, renamed: "moveUpdate(from:count:)")
```

Comment:
These should also get the spi attribute spelling you came up with. (Later)

Reply:
That won't work well for deprecations, where these must remain public.

Reply:
Ah, yes. Correct.